src/components/RightSidebar/LobbyStatus.vue

```diff
@@ -20,17 +20,20 @@
 -->
 
 <template>
-	<button @click="disableLobby">
+	<Button type="secondary" @click="disableLobby">
```

Comment:
Throws an error until secondary is added to the validator:
Fixed upstream at https://github.com/nextcloud/nextcloud-vue/pull/2540

Reply:
https://github.com/nextcloud/nextcloud-vue/pull/2540
I can see it was merged just before, then do I need to remove type in this button? :)

Reply:
No keep the type. But there is an error in the console log, until vue library was updated to 5.1.1

Reply:
Ok

Reply:
Since this is actually a rather important button and you have to press it before you can start your meeting, should we make it primary? @marcoambrosini

Reply:
yes sounds good